src/Adapter/Security/SslMiddleware.php

```diff
@@ -27,7 +27,12 @@
 namespace PrestaShop\PrestaShop\Adapter\Security;
 
 use Configuration;
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use PrestaShopBundle\Controller\Api\OAuth2\AccessTokenController;
```

Comment:
This dependency is weird.  The main reason is because this file is misplaced, this file should not be in `src/Adapter` but in src/PrestaShopBundle.

Do you think we can move it , or deprecate the one into src/Adapter ?

Reply:
Why would a controller be in `src/Adapter`? 🤔
We're just checking that the controller called is part of the API so we check the TLS version.